ui-tests/src/test/resources/source-view/SampleClassWithoutPackage.java

```diff
@@ -0,0 +1,3 @@
+public class Main {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All classes, interfaces, enums and annotations must belong to a named package](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5070362)